# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Add new Catalan locales
# DP: Related bugs: BZ2158
# DP: Dpatch author: Robert Millan  <robertmh@gnu.org>
# DP: Patch author: Robert Millan  <robertmh@gnu.org>
# DP: Upstream status: BZ2158
# DP: Date: 2006-01-16

2006-01-17  Robert Millan  <robertmh@gnu.org>

	* localedata/SUPPORTED: Add ca_AD, ca_FR and ca_IT variants.
	* localedata/locales/ca_AD: New.  Catalan for Andorra.
	* localedata/locales/ca_FR: New.  Catalan for France.
	* localedata/locales/ca_IT: New.  Catalan for Italy.

Index: glibc-2.3.5/localedata/SUPPORTED
===================================================================
--- glibc-2.3.5.orig/localedata/SUPPORTED
+++ glibc-2.3.5/localedata/SUPPORTED
@@ -60,9 +60,15 @@
 bs_BA.UTF-8/UTF-8 \
 bs_BA/ISO-8859-2 \
 byn_ER/UTF-8 \
+ca_AD.UTF-8/UTF-8 \
+ca_AD/ISO-8859-15 \
 ca_ES.UTF-8/UTF-8 \
 ca_ES/ISO-8859-1 \
 ca_ES@euro/ISO-8859-15 \
+ca_FR.UTF-8/UTF-8 \
+ca_FR/ISO-8859-15 \
+ca_IT.UTF-8/UTF-8 \
+ca_IT/ISO-8859-15 \
 cs_CZ.UTF-8/UTF-8 \
 cs_CZ/ISO-8859-2 \
 cy_GB.UTF-8/UTF-8 \
Index: glibc-2.3.5/localedata/locales/ca_AD
===================================================================
--- /dev/null
+++ glibc-2.3.5/localedata/locales/ca_AD
@@ -0,0 +1,94 @@
+comment_char %
+escape_char  /
+%
+% Catalan Language Locale for Andorra 
+% Language: ca
+% Territory: AD
+% Option: euro
+% Revision: 1.0
+% Date: 2006-01-16
+% Application: general
+% Users: general
+% Charset: ISO-8859-15
+% Distribution and use is free, also
+% for commercial purposes.
+
+LC_IDENTIFICATION
+title      "Catalan locale for Andorra "
+source     "Robert Millan"
+address    ""
+contact    ""
+email      ""
+tel        ""
+fax        ""
+language   "Catalan"
+territory  "Andorra"
+revision   "1.0"
+date       "2006-01-16"
+%
+category  "ca_AD:2000";LC_IDENTIFICATION
+category  "ca_AD:2000";LC_CTYPE
+category  "ca_AD:2000";LC_COLLATE
+category  "ca_AD:2000";LC_TIME
+category  "ca_AD:2000";LC_NUMERIC
+category  "ca_AD:2000";LC_MONETARY
+category  "ca_AD:2000";LC_MESSAGES
+category  "ca_AD:2000";LC_PAPER
+category  "ca_AD:2000";LC_NAME
+category  "ca_AD:2000";LC_ADDRESS
+category  "ca_AD:2000";LC_TELEPHONE
+
+END LC_IDENTIFICATION
+
+LC_COLLATE
+copy "ca_ES"
+END LC_COLLATE
+
+LC_CTYPE
+copy "i18n"
+END LC_CTYPE
+
+LC_MESSAGES
+copy "ca_ES"
+END LC_MESSAGES
+
+LC_MONETARY
+copy "ca_ES"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "ca_ES"
+END LC_NUMERIC
+
+LC_TIME
+copy "ca_ES"
+END LC_TIME
+
+LC_PAPER
+copy "ca_ES"
+END LC_PAPER
+
+LC_TELEPHONE
+tel_int_fmt    "<U002B><U0025><U0063><U0020><U0025><U0061><U0020><U0025>/
+<U006C>"
+int_prefix     "<U0033><U0037><U0036>"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "ca_ES"
+END LC_MEASUREMENT
+
+LC_NAME
+copy "ca_ES"
+END LC_NAME
+
+LC_ADDRESS
+postal_fmt    "<U0025><U0066><U0025><U004E><U0025><U0061><U0025><U004E>/
+<U0025><U0064><U0025><U004E><U0025><U0062><U0025><U004E><U0025><U0073>/
+<U0020><U0025><U0068><U0020><U0025><U0065><U0020><U0025><U0072><U0025>/
+<U004E><U0025><U0025><U007A><U0020><U0025><U0054><U0025>/
+<U004E><U0025><U0063><U0025><U004E>"
+country_ab2 "<U0041><U0044>"
+country_ab3 "<U0041><U004E><U0044>"
+country_num 20
+END LC_ADDRESS
Index: glibc-2.3.5/localedata/locales/ca_FR
===================================================================
--- /dev/null
+++ glibc-2.3.5/localedata/locales/ca_FR
@@ -0,0 +1,85 @@
+comment_char %
+escape_char  /
+%
+% Catalan Language Locale for France 
+% Language: ca
+% Territory: FR
+% Option: euro
+% Revision: 1.0
+% Date: 2006-01-16
+% Application: general
+% Users: general
+% Charset: ISO-8859-15
+% Distribution and use is free, also
+% for commercial purposes.
+
+LC_IDENTIFICATION
+title      "Catalan locale for France "
+source     "Robert Millan"
+address    ""
+contact    ""
+email      ""
+tel        ""
+fax        ""
+language   "Catalan"
+territory  "France"
+revision   "1.0"
+date       "2006-01-16"
+%
+category  "ca_FR:2000";LC_IDENTIFICATION
+category  "ca_FR:2000";LC_CTYPE
+category  "ca_FR:2000";LC_COLLATE
+category  "ca_FR:2000";LC_TIME
+category  "ca_FR:2000";LC_NUMERIC
+category  "ca_FR:2000";LC_MONETARY
+category  "ca_FR:2000";LC_MESSAGES
+category  "ca_FR:2000";LC_PAPER
+category  "ca_FR:2000";LC_NAME
+category  "ca_FR:2000";LC_ADDRESS
+category  "ca_FR:2000";LC_TELEPHONE
+
+END LC_IDENTIFICATION
+
+LC_COLLATE
+copy "ca_ES"
+END LC_COLLATE
+
+LC_CTYPE
+copy "i18n"
+END LC_CTYPE
+
+LC_MESSAGES
+copy "ca_ES"
+END LC_MESSAGES
+
+LC_MONETARY
+copy "ca_ES"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "ca_ES"
+END LC_NUMERIC
+
+LC_TIME
+copy "ca_ES"
+END LC_TIME
+
+LC_PAPER
+copy "fr_FR"
+END LC_PAPER
+
+LC_TELEPHONE
+copy "fr_FR"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "fr_FR"
+END LC_MEASUREMENT
+
+LC_NAME
+copy "ca_ES"
+END LC_NAME
+
+LC_ADDRESS
+copy "fr_FR"
+END LC_ADDRESS
Index: glibc-2.3.5/localedata/locales/ca_IT
===================================================================
--- /dev/null
+++ glibc-2.3.5/localedata/locales/ca_IT
@@ -0,0 +1,85 @@
+comment_char %
+escape_char  /
+%
+% Catalan Language Locale for Italy (L'Alguer) 
+% Language: ca
+% Territory: IT
+% Option: euro
+% Revision: 1.0
+% Date: 2006-01-16
+% Application: general
+% Users: general
+% Charset: ISO-8859-15
+% Distribution and use is free, also
+% for commercial purposes.
+
+LC_IDENTIFICATION
+title      "Catalan locale for Italy (L'Alguer) "
+source     "Robert Millan"
+address    ""
+contact    ""
+email      ""
+tel        ""
+fax        ""
+language   "Catalan"
+territory  "Italy (L'Alguer)"
+revision   "1.0"
+date       "2006-01-16"
+%
+category  "ca_IT:2000";LC_IDENTIFICATION
+category  "ca_IT:2000";LC_CTYPE
+category  "ca_IT:2000";LC_COLLATE
+category  "ca_IT:2000";LC_TIME
+category  "ca_IT:2000";LC_NUMERIC
+category  "ca_IT:2000";LC_MONETARY
+category  "ca_IT:2000";LC_MESSAGES
+category  "ca_IT:2000";LC_PAPER
+category  "ca_IT:2000";LC_NAME
+category  "ca_IT:2000";LC_ADDRESS
+category  "ca_IT:2000";LC_TELEPHONE
+
+END LC_IDENTIFICATION
+
+LC_COLLATE
+copy "ca_ES"
+END LC_COLLATE
+
+LC_CTYPE
+copy "i18n"
+END LC_CTYPE
+
+LC_MESSAGES
+copy "ca_ES"
+END LC_MESSAGES
+
+LC_MONETARY
+copy "ca_ES"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "ca_ES"
+END LC_NUMERIC
+
+LC_TIME
+copy "ca_ES"
+END LC_TIME
+
+LC_PAPER
+copy "it_IT"
+END LC_PAPER
+
+LC_TELEPHONE
+copy "it_IT"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "it_IT"
+END LC_MEASUREMENT
+
+LC_NAME
+copy "ca_ES"
+END LC_NAME
+
+LC_ADDRESS
+copy "it_IT"
+END LC_ADDRESS
